spharpy/spherical.py

```diff
@@ -767,3 +768,404 @@ def sph_identity_matrix(n_max, type='n-nm'):
         identity_matrix[n, linear_nm] = 1
 
     return identity_matrix
+
+
+
+class SphericalHarmonics:
+    r"""
+    Compute spherical harmonic basis matrices, their inverses, and gradients.
+    The spherical harmonic Ynm is given by:
+    .. math::
+        Y_{nm} = N_{nm} P_{nm}(cos(\theta)) T_{nm}(\phi)
+
+    where:
+    - $n$ is the degree
```

Comment:
```suggestion
    where:
 
    - :math:`n` is the degree
```
in this was the bulletpoints and math would be fixed